client/packages/programs/src/JsonForms/components/BloodPressure.tsx

```diff
@@ -28,18 +27,7 @@ export const UIComponent = (props: ControlProps) => {
     BloodPressureData | undefined
   >(data);
 
-  const { customError, setCustomError } = useJSONFormsCustomError(
-    path,
-    'BloodPressure'
-  );
-
-  useEffect(() => {
-    if (core) {
-      const getChildErrors = subErrorsAt(path, schema);
-      const errors = getChildErrors(core);
-      setCustomError(errors[0]?.message);
-    }
-  }, [core]);
```

Comment:
This is the nasty `useEffect` I mentioned above. It wouldn't play nicely at all with the custom errors and the child errors both changing, so it's easier to remove it completely. This means that *only* the child data nodes are in "error" state, but I think that's fine as it's technically more correct. Just means we need that slight relaxation of the "matching" in the Category tile components.